skewed/src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+class Header extends Component {
+  render() {
+    return (
+      <div className="header">
+      <h1> The Skewed Ratio Project </h1>
```

Comment:
Indent this. Install eslint and copy the .eslintrc file from MBS to help you get started